channels/us.m3u

```diff
@@ -3691,9 +3687,9 @@ http://wpso.com:1936/hls/wzra.m3u8
 https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8
 #EXTINF:-1 tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living
 https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r_prod/tracks-v1a1/index.m3u8
-#EXTINF:-1 tvg-id="ZooMoo.us" tvg-name="Zoo Moo" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/1wYnA2i.jpg" group-title="",Zoo Moo (720p)
+#EXTINF:-1 tvg-id="ZooMoo.us" tvg-name="Zoo Moo" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/1wYnA2i.jpg" group-title="Kids",Zoo Moo (720p)
 https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="KanalDisney.us" tvg-name="Канал Disney" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Kids",Канал Disney (576p)
 http://188.40.68.167/russia/disney/playlist.m3u8
-#EXTINF:-1 tvg-id="MeiGuoQieDaDianShi.us" tvg-name="美国切达电视" tvg-country="US" tvg-language="" tvg-logo="" group-title="",美国切达电视 (1080p)
+#EXTINF:-1 tvg-id="CheddarNews.us" tvg-name="Cheddar News" tvg-country="US" tvg-language="English" tvg-logo="https://miro.medium.com/max/2160/1*nrRFfv8W3RuFZQpskTIH3A.jpeg" group-title="News",Cheddar News (1080p)
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
```

Comment:
Not sure about that one. Is it the English Cheddar News ?

Reply:
yes. tested link and that is whats pulled

Reply:
Tested again. it is cheddar news

Reply:
Doesn't even seem to work for me. I'm only getting advertisments for now.

Reply:
tested again to triple check. comes up the same

Reply:
glad it wasnt just me